File: Stage2/Java/Problem2753.java
package Stage2.Java;

import java.util.Scanner;

public class Problem2753 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int year = sc.nextInt();

        boolean leapYear = (year % 4 ==0 && year % 100 != 0) || (year % 400 == 0);

        System.out.println(leapYear ? 1 : 0);

        sc.close();
    }
}

File: Stage2/C/Problem2753.c
#include <stdio.h>
#include <stdbool.h>

int main() {
    int year;
    scanf("%d", &year);
    bool leapYear = (year % 4 == 0 && year % 100 != 0) || (year % 400 == 0);
    printf("%d\n", leapYear ? 1 : 0);

    return 0;
}